openwisp_controller/geo/api/urls.py

```diff
@@ -1,6 +1,24 @@
-from ..utils import get_geo_urls
+from django.urls import path
+
 from . import views as geo_views
 
 app_name = 'openwisp_controller'
 
+
+def get_geo_urls(geo_views):
+    return [
+        path(
+            'api/v1/device/<str:pk>/location/',
+            geo_views.device_location,
+            name='device_location',
+        ),
+        path('api/v1/location/geojson/', geo_views.geojson, name='location_geojson',),
```

Comment:
Please format this in the same way as the lines above and below.

Reply:
I think the formatting is not a problem here, it's just github showing it this way.
If you check it on local, it's it one line only.

Reply:
For me, it's locally the same as it is displayed by Github. The second `path` statement is on one line whereas the other two `path` statements are spread across several lines. And it's the same here: https://github.com/openwisp/openwisp-controller/pull/474/commits/9150658a168467076e80c5aee85efda16b3c0ddc#diff-aba8a6bb462758a58d3fe894f46b949b6b5c5868abc32f620d8edd2e5bf2845bR11

Reply:
Yes, It's in one line because it's less than 88 characters. (or whatever was the limit)
`black` is taking care of all the formatting, CI would fail if the line extend a certain allowed limit. :smile:

Reply:
@atb00ker Thank you for the explanation. It's ok if it follows the rules, it's only that the human eye is still irritated if similar lines don't have the same format - pattern recognition :grinning: